src/platform_impl/android/mod.rs

```diff
@@ -43,6 +44,7 @@ fn poll(poll: Poll) -> Option<EventSource> {
 
 pub struct EventLoop<T: 'static> {
     window_target: event_loop::EventLoopWindowTarget<T>,
+    native_window_lock: Option<RwLockReadGuard<'static, Option<NativeWindow>>>,
```

Comment:
```suggestion
    /// Hold a readlock on the native window between `Resumed` and `Suspended` due to/because [..]
    native_window_lock: Option<RwLockReadGuard<'static, Option<NativeWindow>>>,
```

This file is quite lacking documentation wise, perhaps this little intricacy is a great place to start changing that!

Reply:
Sure!